app/src/main/java/com/example/teamyuml/decentworkmobile/Login.java

```diff
@@ -1,4 +1,136 @@
+import android.widget.Button;
+import android.widget.EditText;
+import android.content.Intent;
+import android.view.View;
+import android.widget.Toast;
+import com.android.volley.Request;
+import com.android.volley.Response;
+import com.android.volley.VolleyError;
+import com.android.volley.toolbox.StringRequest;
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.util.HashMap;
+import java.util.Map;
+
+/*
+ * Class creating connection with server
+ * Checking adres email
+ * Parsing data to JSON and sending to the server
+ */
+
+public class Login extends AppCompatActivity {
+
+
+    ProgressDialog progressDialog;
+    private EditText loginInputEmail, loginInputPassword;
+    private Button btnlogin;
+    private Button btnLinkSignup;
+    private static final String TAG = "Login";
+    private static final String URL_FOR_LOGIN = "Heve we wanna url adress for Login";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        this.requestWindowFeature(Window.FEATURE_NO_TITLE);
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+        loginInputEmail = (EditText) findViewById(R.id.login_input_email);
+        loginInputPassword = (EditText) findViewById(R.id.login_input_password);
+        btnlogin = (Button) findViewById(R.id.btn_login);
+        btnLinkSignup = (Button) findViewById(R.id.btn_link_signup);
+        // Progress dialog
+        progressDialog = new ProgressDialog(this);
+        progressDialog.setCancelable(false);
+
+        btnlogin.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                loginUser(loginInputEmail.getText().toString(),
+                        loginInputPassword.getText().toString());
+            }
+        });
+
+        btnLinkSignup.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Intent i = new Intent(getApplicationContext(), Registration.class);
+                startActivity(i);
+            }
+        });
+    }
+
+    private void loginUser( final String email, final String password) {
+        // Tag used to cancel the request
+        String cancel_req_tag = "login";
+        progressDialog.setMessage("Logging you in...");
+        showDialog();
+        StringRequest strReq = new StringRequest(Request.Method.POST,
+                URL_FOR_LOGIN, new Response.Listener<String>() {
+
+            @Override
+            public void onResponse(String response) {
+                Log.d(TAG, "Register Response: " + response.toString());
+                hideDialog();
+                try {
+                    JSONObject jObj = new JSONObject(response);
+                    boolean error = jObj.getBoolean("error");
+
+                    if (!error) {
+                        String user = jObj.getJSONObject("user").getString("name");
```

Comment:
Link with possible responses from server to the url:
https://drive.google.com/open?id=1fRbLmRcEcXfW3E1WavMrksxNZtTCFNjaz8ISEd6tgIA